socrata/socrata_upload.py

```diff
@@ -172,31 +176,56 @@ def upload(method, asset_id, sql_query, overwrite, year=None, township=None):
     input_data[date_columns] = input_data[date_columns].map(
         lambda x: x.strftime("%Y-%m-%dT%X")
     )
-    input_data = input_data.to_json(orient="records")
 
     # Raise URL status if it's bad
-    requests.get(
-        url=url,
-        data=input_data,
-        auth=auth,
+    s.get(
+        (
+            "https://datacatalog.cookcountyil.gov/resource/"
+            + asset_id
+            + ".json?$limit=1"
+        ),
+        headers={"X-App-Token": os.getenv("SOCRATA_APP_TOKEN")},
     ).raise_for_status()
 
-    print(print_message)
-    if method == "put":
-        response = requests.put(
-            url=url,
-            data=input_data,
-            auth=auth,
-        )
+    s.get(url=url).raise_for_status()
+
+    if input_data.shape[0] > 10000:
+        for i in range(0, input_data.shape[0], 10000):
+            print(print_message)
+            print(f"Rows {i + 1}-{i + 10000}")
+            if method == "put":
+                response = s.put(
+                    url=url,
+                    data=input_data.iloc[i : i + 10000].to_json(
+                        orient="records"
+                    ),
+                )
+
+            elif method == "post":
+                response = s.post(
+                    url=url,
+                    data=input_data.iloc[i : i + 10000].to_json(
+                        orient="records"
+                    ),
+                )
 
-    elif method == "post":
-        response = requests.post(
-            url=url,
-            data=input_data,
-            auth=auth,
-        )
+            print(response.content)
```

Comment:
**[Question, non-blocking]** What sort of content do we expect to get back from the response? I just want to make absolutely sure that it can't contain anything sensitive.